app/ai/voice/agents/breeze_buddy/template/loader.py

```
@@ -205,13 +211,30 @@ async def load_template(
                     )
 
+        # 5. Inject data_source content.
+        if template_obj.data_sources:
+            contents = await asyncio.gather(
+                *[
+                    self._fetch_one_data_source(ref)
+                    for ref in template_obj.data_sources
+                ],
+                return_exceptions=True,
+            )
+            for ref, result in zip(template_obj.data_sources, contents):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)